html/pfappserver/root/static.alt/src/components/pfNetworkGraphTooltipNode.vue

```diff
@@ -1,22 +1,18 @@
 <template>
-  <b-card no-body>
+  <b-card no-body class="pf-network-graph-tooltip-node">
     <b-card-header>
-      <h5 class="mb-0"><mac>{{ id }}</mac></h5>
+      <h5 class="mb-0 text-nowrap">{{ $t('Node') }}</h5>
+      <p class="mb-0"><mac>{{ id }}</mac></p>
     </b-card-header>
-    <div class="card-body">
-      <b-container class="my-3 px-0" v-if="isLoading || isError">
+    <div class="card-body" v-if="isLoading || !isError">
+      <b-container class="my-3 px-0" v-if="isLoading">
         <b-row class="justify-content-md-center text-secondary">
-            <b-col cols="12" md="auto" class="w-100">
-              <icon name="circle-notch" scale="2" spin v-if="isLoading"></icon>
-              <b-media v-else>
-                <icon name="exclamation-triangle" scale="2" slot="aside"></icon>
-                <strong v-t="'Node'"></strong>
-                <p class="text-nowrap" v-t="'Not found'"></p>
-              </b-media>
-            </b-col>
+          <b-col cols="12" md="auto" class="w-100 text-center">
+            <icon name="circle-notch" scale="2" spin></icon>
+          </b-col>
         </b-row>
       </b-container>
-      <b-container fluid class="px-0" v-else>
+      <b-container fluid class="container px-0" v-else-if="!isError">
```

Comment:
No need to use a *fluid* container for tooltips. I would simply use `<b-container class="px-0" ..>`.